src/overlays/actors/ovl_En_Anubice/z_en_anubice.c

```diff
@@ -358,6 +358,7 @@ void EnAnubice_Update(Actor* thisx, GlobalContext* globalCtx) {
         this->actor.shape.yOffset == 0.0f) {
         EnAnubice_Hover(this, globalCtx);
         for (i = 0; i < 5; i++) {
```

Comment:
This seems like a bug, because this->flameCircles only has length 4, or the loop must somehow always be broken before it runs off the end of the array. Or is the declaration just wrong?

Reply:
That makes sense. As far as I can see, the code never uses more than 5 elements of flameCircle. That still leaves 4 bytes of unused space after it, though. Maybe in the original code, it was an array of length 6, where the last element was unused?